inc/spbc-settings.php

```diff
@@ -317,18 +317,18 @@ function spbc_settings__register() {
 						'type' => 'section',
 						'title' => __('Firewall', 'security-malware-firewall'),
 						'display' => $spbc->fw_enabled,
-						'description' => __('Plugin will automatically add IP address for each session with administration rights to Personal list in the cloud.', 'security-malware-firewall'),
+						'description' => __('The plugin will automatically add an IP address of your website visitor to each session and give you the link with administrator rights to edit your Personal Lists in the CleanTalk Cloud.', 'security-malware-firewall'),
```

Comment:
Any IP addresses of the logged in administrators will be automatically added to your Personal Lists and will be approved all the time.

Reply:
Fixed.